pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -361,6 +361,13 @@ func (lb *ListenerBuilder) buildHTTPConnectionManager(httpOpts *httpListenerOpts
 		Port:  httpOpts.port,
 		Class: httpOpts.class,
 	})
+
+	// Metadata exchange filter needs to be added before any other HTTP filters are added. This is done to
+	// ensure that mx filter comes before HTTP RBAC filter. This is related to https://github.com/tetrateio/tetrate/issues/13093
```

Comment:
tetrate one is private, can we just link to the istio one?

Reply:
Done, thanks!